utils/posts.ts

```diff
@@ -0,0 +1,37 @@
+import { extract } from "std/encoding/front_matter/yaml.ts";
```

Comment:
```suggestion
import { extract } from "std/front_matter/yaml.ts";
```
Note: `std/encoding/front_matter` is now deprecated and will soon be removed.